doc/rados/operations/crush-map.rst

```diff
@@ -326,214 +326,204 @@ Although weight sets can be set up and adjusted manually, we recommend enabling
 the ``ceph-mgr`` *balancer* module to perform these tasks automatically if the
 cluster is running Luminous or a later release.
 
-
 Modifying the CRUSH map
 =======================
 
 .. _addosd:
 
-Add/Move an OSD
----------------
+Adding/Moving an OSD
+--------------------
 
-.. note: OSDs are normally automatically added to the CRUSH map when
-         the OSD is created.  This command is rarely needed.
+.. note:: Normally, OSDs automatically add themselves to the CRUSH map when
```

Comment:
Two adverbs seems like overkill, I would s/Normally, //

Reply:
@zdover23 - Maybe solve the adverb overexposure by replacing _Normally_ with _Under normal conditions_.

Reply:
Accepted.